tests/testthat/test-stan_expected_obs.R

```diff
@@ -3,11 +3,66 @@ skip_on_os("windows")
 skip_on_os("mac")
 skip_on_local()
 
-enw_stan_to_r(
-  c("expected_obs.stan", "hazard.stan"),
-  system.file("stan", "functions", package = "epinowcast")
-)
-# note these tests require enw_stan_to_r() to be run first
```

Comment:
Why remove this bit?

Reply:
In this case should we also remove it from the discretized logit hazard test file?

Reply:
yes good spot